src/Util/Argument.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Friendica\Util;
+
+use Friendica\Network\HTTPException\InternalServerErrorException;
+
+class Argument
+{
+	/**
+	 * Get a value to a given key in the arguments
+	 *
+	 * @param string $key     The search key
+	 * @param array  $args    The arguments to search
+	 * @param string $default The default value if no value was found (default empty)
+	 * @param string $type    The type of value it has to be (default null = all)
+	 *
+	 * @return string The value to the key
+	 *
+	 * @throws InternalServerErrorException Arguments are not valid
+	 */
+	public static function get($key, $args, $default = '', $type = null) {
```

Comment:
Now that I'm looking at it in isolation, I feel like much is wrong with this class. If I understand clearly it's used to convert from command line arguments to a hash and vice-versa.

Therefore, I think it should be an actual object instantiated with the `$_SERVER['argv']` value by default or an array of arguments. You can even throw 

Suggested use case:
```php
$arguments = new CLIArguments(); // assuming $_SERVER['argv']
//or
$arguments = new CLIArguments($argv);

$arguments->get('key', $default, $type);
$arguments->set('newkey', $value);
// Returns a string of command line arguments
$args = $arguments->toString();

$command .= '/usr/bin/php ' . $args;
```

Additionally, you have to make sure that the returned command line argument string works with the current OS (see usage in your src/App.php:1134).

You can even throw `ArrayAccess` for good measure! See http://php.net/manual/en/class.arrayaccess.php

```php
class CLIArguments implements ArrayAccess {
    ...
}

$arguments = new CLIArguments(); // assuming $_SERVER['argv']
//or
$arguments = new CLIArguments($argv);

$value = $arguments['key'];
$arguments['newkey'] = $value;

// Returns a string of command line arguments
$args = $arguments->toString();

$command .= '/usr/bin/php ' . $args;
```

Reply:
Thank you @MrPetovan !
That's why I want to split the big PR in different smaller PRs.

It looks much better than the static class :-)